.eslintrc

```diff
@@ -1,7 +1,40 @@
 {
-  "extends": "@ahmdigital/eslint-config/react-config",  
-  "parser": "@typescript-eslint/parser",
+  "root": true,
+  "env": {
+    "node": true,
+    "browser": true
+  },
+  "extends": [
+    "@ahmdigital/eslint-config/react-config",
```

Comment:
Can/should we greater leverage the eslint config repo? (notably for typescript like we do in sales repo)